### ISSUE TITLE
Can VTK.js slice on the polydata? 

### ISSUE THREAD

**Issue by chiyandetaotie** [Kitware/vtk-js#3010]
### Feature Request

Can VTK.js slice on the polydata? I want to slice cfd data with vtk.js on the browser to show post-processing result. How can I do?

### Motivation and Detailed Description

Can VTK.js slice on the polydata? I want to slice cfd data with vtk.js on the browser to show post-processing result. How can I do?

**Comment by floryst** [comment]
Yes, see the [vtkCutter example](https://kitware.github.io/vtk-js/examples/Cutter.html).

**Comment by chiyandetaotie** [comment]
I want to see the grid on the slice. How can I do?
